tests/basic.t

```diff
@@ -65,49 +65,94 @@ if [ $? -eq 1 ]; then
   TEST glusterd
 fi
 
-# Create gluster volume
+##### Create Gluster Volume Test Start #####
 TEST gluster vol create ${VOLNAME} ${HOST}:${BRKDIR} force
 
 # Start the volume
 TEST gluster vol start ${VOLNAME}
+##### End #####
 
 
-# Test gfapi access
+##### Gfapi Access Test Start #####
 TEST ./tests/gfapi-test ${VOLNAME} ${HOST}
+##### End #####
 
-# Start gluster-blockd.service
+
+##### Restart gluster-blockd.service Test Start #####
 systemctl daemon-reload
-TEST systemctl restart gluster-blockd.service
-sleep 1;
+TEST systemctl restart gluster-block-target.service gluster-blockd.service
+sleep 5;
+##### End #####
+
+
+##### Block 'create' and 'delete' Test Start #####
+# Simple block create/delete
+TEST gluster-block create ${VOLNAME}/${BLKNAME} ${HOST} 1MiB
+TEST gluster-block delete ${VOLNAME}/${BLKNAME}
+
+# Block create with 'auth' set/delete
+TEST gluster-block create ${VOLNAME}/${BLKNAME} auth enable ${HOST} 1MiB
+TEST gluster-block delete ${VOLNAME}/${BLKNAME}
+
+# Block create with 'ha' set/delete
+TEST gluster-block create ${VOLNAME}/${BLKNAME} ha 1 ${HOST} 1MiB
+TEST gluster-block delete ${VOLNAME}/${BLKNAME}
+
+# Block create with 'prealloc' set/delete
+TEST gluster-block create ${VOLNAME}/${BLKNAME} prealloc full ${HOST} 1MiB
+TEST gluster-block delete ${VOLNAME}/${BLKNAME}
```

Comment:
as prealloc is full now by default, should we also test for prealloc no ?

Reply:
Ah, yeah, right, I forgot that changes.
Will add this.